chemprop/__init__.py

```diff
@@ -16,4 +16,6 @@
 import chemprop.spectra_utils
 import chemprop.hyperopt_utils
 
-from chemprop._version import __version__
+# from chemprop._version import __version__
+
+__version__ = "1.4.1"
```

Comment:
We should have a newline at the end right?

Reply:
i guess that would be proper